src/main/java/decodes/datasource/PMParser.java

```diff
@@ -118,6 +118,8 @@ public static PMParser getPMParser(String headerType)
 
 		// First use enum to lookup headerType, retrieve class
 		// name. Then instantiate the class.
+		if (Database.getDb() == null || Database.getDb().enumList == null)
```

Comment:
Might be better to throw an exception. If a null return is acceptable change to Optional<PMParser> (if the work doesn't snowball out of control)

Also, needs { }.